.travis.yml

```diff
@@ -1,10 +1,17 @@
 language: python
 
-python:
-  - '3.6'
+group: travis_latest
```

Comment:
This moves away from the stable lts, and to the more feature filled latest image. A little more info [here](https://blog.travis-ci.com/2017-12-01-new-update-schedule-for-linux-build-images).